src/views/base.ts

```diff
@@ -24,14 +24,24 @@ export abstract class BaseWebviewViewProvider<TViewMessage extends Message, TPro
         webviewView.webview.onDidReceiveMessage((data: TViewMessage) => {
             this.handleMessage(data);
         });
+
+        // Call onViewReady if it exists (for subclasses that implement it)
+        if ('onViewReady' in this && typeof (this as unknown as { onViewReady?: () => void | Promise<void> }).onViewReady === 'function') {
+            void (this as unknown as { onViewReady: () => void | Promise<void> }).onViewReady();
+        }
```

Comment:
Cleaner (untested) suggestion:
```suggestion
        // Call onViewReady if it exists (for subclasses that implement it)
        const maybeViewReady = this as { onViewReady?: () => void | Promise<void> };
        if (typeof maybeViewReady.onViewReady === 'function') {
            void maybeViewReady.onViewReady();
        }
```